web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -193,6 +122,107 @@ fun WebView(
     }
 }
 
+/**
+ * AccompanistWebViewClient
+ *
+ * A parent class implementation of WebViewClient that can be subclassed to add custom behaviour.
+ *
+ * As Accompanist Web needs to set its own web client to function, it provides this intermediary
+ * class that can be overriden if further custom behaviour is required.
+ */
+open class AccompanistWebViewClient : WebViewClient() {
+    internal lateinit var state: WebViewState
```

Comment:
Make these two `open` so that they can be intercepted by another impl?

Reply:
I really don't want people to be able to go `webViewClient.state = someOtherState` when using it so maybe
```
open lateinit var state: WebViewState
        internal set
```